### ISSUE TITLE
bug/discover-artists-page: Colour contrast of links (visited and unvisited) and background too low

### ISSUE THREAD

**Issue by KathyPeacock** [dylanlittle/team-moshpit-project#170]
### Severity

High

### Suggested Priority

High

### Observed Behaviour

The link text is very hard to read in blue (or purple when visited) against the dark background. This affects general impression and universal usability of the site, and is an accessibility concern. (Stands out especially in links such as screenshotted below on the 'Discover Artists' page - especially considering the majority of the text styling is a really effective white against black)

<img width="705" height="525" alt="Image" src="https://github.com/user-attachments/assets/730249de-e563-4e57-b323-d3ea043c04ea" />

### Expected Behaviour

Colour contrast between background and any text should be as distinct as possible. Pale colours that stand out clearly are preferable, and the change of colour when a link has been visited - or hover/click interaction colour change should be taken into consideration in the same way.

### Reproduction Steps

n/a
